docs/networking/a-quick-guide-to-ip-addressing.md

```diff
@@ -4,48 +4,21 @@ title: A quick guide to IP addressing
-any TitanNet devices you are using and any Art-Net fixtures. Beware that
-some Art-Net fixtures have a fixed IP address in the range `2.x.x.x`, in
-which case you have to set the console to that range.
-
-Subnet Masks
-------------
-
-These are best kept simple. All devices or nodes on a network who wish
-to communicate with each other must have the same subnet mask. The mask
-determines which parts of the IP address are unique in that network to
-each node. If the part of the subnet mask is a 0 then the corresponding
-digit in an IP address must be unique for each fixture. If the part in a
-subnet mask is 255 then this part of the IP address needs to be the same
-for each node.
+Every device on a network must have a unique IP address. IP addresses have the format `w.x.y.z` where `w`, `x`, `y` and `z` are numbers between 0 and 255, for example `192.168.0.1`. The address can be set manually (this is called a **Static** IP address) or can be automatically allocated by the network (using a system called **DHCP**). In lighting networks static addressing is normally used, it takes a bit more time to set up but you then know for sure what the address is for each device.
+
+For devices to be able to "see" each other on the network, they must be in the same **Subnet** - this is the first part of the IP address. Each device has a **Subnet Mask** which sets the parts of the IP address which must match for the devices to be able to see each other, and which parts must be different. 
+
+Subnet Masks are often set to `255.255.255.0` which means that the `w` `x` and `y` numbers must match but the `z` number must be different. So if the console was set at `192.168.1.1` then the subnet would be `192.168.1.z` and all the other devices would be `192.168.1.2`, `192.168.1.3` and so on.
+
+If your IP address is allocated by DHCP then the Subnet Mask is set automatically. Subnet masks are sometimes referred to as `/24` or `/8`, this is the number of bits set to 1 in the mask. Each number in the mask is 8 bits so `255.255.255.0` can also be called `/24`, or `255.0.0.0.` would be `/8`.
+
 
 Choosing an IP address and Subnet Mask
 --------------------------------------
 
-This is the hardest part of setting up a network as your IP address
-totally depends on what you are using on the network and what IP
-addresses you can and cannot change. Below are a number of example
-scenarios for standard lighting networks using a Titan console and what
-IP addresses should be set. *These aren't guaranteed to work but try them
-if the scenario matches your network.*
+This is the hardest part of setting up a network as a suitable IP address totally depends on what you are using on the network and what IP addresses you can and cannot change. Some older Art-Net equipment is fixed to the address range `2.x.y.z` or `10.x.y.z` which means everything else has to use that range as well. But if none of your equipment is fixed, the address range `192.168.1.x` is often used.
```

Comment:
192.168.0.0/16, the 1 doesn't need to be fixed

Reply:
No I don't think it really matters given the above example also uses 192.168.1.1